start_bundle_coupled.R

```diff
@@ -435,14 +407,15 @@ for(scen in common){
         path_report <- path_report_found
       }
     } else {
+      # start_coupled.R uses the name of this run to create the name of the MAgPIE report for the subsequent runs (next iteration)
```

Comment:
As I said, I would also be fine replacing it with the filename it will get later, as it is already known. But this is fine like this as well.